docs/data-structures/depth-of-tree.mdx

```diff
@@ -267,4 +267,4 @@ Depth of tree : 3
 ## Credits
 
 - [Satyam Dengre](https://github.com/satyampgt4) for the [C++ implementation](https://github.com/TesseractCoding/NeoAlgo/blob/master/C-Plus-Plus/graphs/Depth_of_tree.cpp)
-- [Sangeeta Mishra](https://github.com/SangeetaMishra) for the [Python implementation](https://github.com/TesseractCoding/NeoAlgo/blob/master/Python/graphs/Depth_Of_Tree.py)
\ No newline at end of file
+- [Sangeeta Mishra](https://github.com/SangeetaMishra) for the [Python implementation](https://github.com/TesseractCoding/NeoAlgo/blob/master/Python/graphs/Depth_Of_Tree.py)
```

Comment:
Why is this file commited?.

Reply:
It got changed by itself when I tried to rebase, but I don't understand the change in both of them. They look similar.

Reply:
@gat786 I've removed this commit now.

Kindly review :)

Reply:
can you unstage this file?

Reply:
It's unstaged now, you can look at "files changed" and it doesn't appear now.